diff --git a/node_modules/athom-api/dist/types.d.ts b/node_modules/athom-api/dist/types.d.ts
index 11f7726..0938113 100644
--- a/node_modules/athom-api/dist/types.d.ts
+++ b/node_modules/athom-api/dist/types.d.ts
@@ -803,7 +803,7 @@ declare class HomeyAPI {
      * @param {string} [token] - The Homey Token to use
      * @returns {HomeyAPI} The HomeyAPI instance
      */
-    static forCurrentHomey(homey?: any, token?: string): HomeyAPI;
+    static forCurrentHomey(homey?: any, token?: string): Promise<HomeyAPI>;
 }
 
 declare namespace HomeyAPI {
